add import-youtube guide inside documentation

LecygneNoir opened the pull request:
Hello,

Here is the guide to use import-youtube
It should works out of the box, I have tested from scratch on a fresh server :-)

Thanks for this very useful tool!

rigelk commented on support/doc/import-youtube.md:
```diff
@@ -0,0 +1,57 @@
+ - YOUTUBE_USER_URL : the youtube channel you want to import. Support Youtube channel (eg https://www.youtube.com/channel/UCu07SkvzNoU4oiIZpIXCExw) or Youtube user (eg: https://www.youtube.com/c/LecygneNoir or https://www.youtube.com/user/LecygneNoir)
+
+ The script will get all public videos from Youtube, download them, then upload to Peertube.  
+ Already downloaded videos will not be uploaded twice, so you can run and re-run the script in case of crash, deconnexion, ... without problem.
```
s/deconnexion/disconnection

rigelk commented on support/doc/import-youtube.md:
```diff
@@ -0,0 +1,57 @@
+
+You need at least 512MB RAM to run the script.  
+Importation can be launched directly from a peertube server (in this case you already have dependencies installed :+1:) or from a separate server, even a dekstop PC.  
+It is also advised to have maximum bandwith possible as you will download and upload video massively :-)
```
- s/bandwith/bandwidth
- it's a tautology :-)

rigelk commented on support/doc/import-youtube.md:
```diff
@@ -0,0 +1,57 @@
+# Import viddeos from Youtube guide
+
+If you already own a Youtube channel, you can use a Peertube scripts to import all your videos inside Peertube.
```
- s/scripts/script
- technically, the script works even if you don't own the YouTube channel posting the video(s)

LecygneNoir replied:
I do not know if it worth specifying that, could'nt we face problem if anybody import YT videos they do not own inside Peertube?

rigelk replied:
Then we ought to rephrase: "You can use this script to import YouTube videos inside PeerTube. Be sure you own the videos or have their author's licence to do so."

rigelk commented on support/doc/import-youtube.md:
````diff
@@ -0,0 +1,57 @@
+node dist/server/tools/import-youtube.js -u "PEERTUBE_URL" -U "PEERTUBE_USER" --password "PEERTUBE_PASSWORD" -y "YOUTUBE_URL"
+```
+
+ - PEERTUBE_URL : the full URL of your peertube server where you want to import, eg: https://videos.lecygnenoir.info
````
We could just say this is the url of the PeerTube serveur we want to upload to with our credentials. Since the end user will require an account on that instance, it should be pretty obvious to them.

Also, if you really feel the need to provide an URL example, I think we should point to `https://peertube.cpy.re/` since it's more or less the official one.

LecygneNoir replied:
Good point, I have use my own links to be more clear, people clicking on it being able to see what is asked (I have some difficulties to found my "youtube user" URL as YT changes their rules about it)
I agree than for Peertube URL, it's totally obvious ^^'

LecygneNoir commented:
Thanks @rigelk for corrections, I re-read 3 times and still let such errors... Shame on me :'(

Chocobozzz commented on support/doc/import-youtube.md:
```diff
@@ -0,0 +1,57 @@
+# Import viddeos from Youtube guide
```
videos

LecygneNoir commented:
Hello,

I have added corrections and placeholder according to your comments, thank you again :-)

I have also added git as dependencies as I have noticed that yarn complains about it on a fresh install.

Hope it helps!

Chocobozzz commented:
Great! Thanks @LecygneNoir!
